addons/point_of_sale/static/src/app/models/data_service.js

```diff
@@ -167,7 +166,8 @@ export class PosData extends Reactive {
             this.setOnline();
             return result;
         } catch (error) {
-            if (queue) {
+            const skipError = error.name == "RPC_ERROR"
```

Comment:
Nice trick, but are you sure that it will not cause side effect ?

Reply:
I couldn't find any, but we could add more condition to the `skipError`. Maybe check that the error is a `ValidationError` ? This might be more safe, what do you think ? @davidmonnom

Reply:
I think we should queue only if there is a network error. Because if there's another error, we can assume it will be repeated.

I don't know if you have any other idea ?

Reply:
Sounds good to me, I'll change it